tiflash/tiflash-configuration.md

```diff
@@ -208,6 +208,12 @@ delta_index_cache_size = 0
     ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
     # dt_enable_logical_split = false
 
+    ## `max_threads`: The default value is the number of CPU cores. This parameter
+    ## is the maximum number of threads executed. This parameter only takes effect
```

Comment:
This parameter is the internal execute concurrency when TiFlash execute a MPP task.